src/node.cc

```diff
@@ -4545,6 +4545,9 @@ inline int Start(Isolate* isolate, IsolateData* isolate_data,
   Environment env(isolate_data, context, v8_platform.GetTracingAgent());
   env.Start(argc, argv, exec_argc, exec_argv, v8_is_profiling);
 
+  TRACE_EVENT_METADATA1("__metadata", "thread_name", "name",
+                        "JavaScriptMainThread");
```

Comment:
Given that trace events can be enabled after this point, perhaps we should move this to ensure that it's always emitted when tracing is first turned on?

Reply:
fwiw... I plan on doing to same to emit some information about which node.js version is being traced